Source/Core/AudioCommon/XAudio2_7Stream.h

```diff
@@ -56,17 +58,24 @@ class XAudio2_7 final : public SoundStream
 	static HMODULE m_xaudio2_dll;
 
 	static bool InitLibrary();
+	u32 samplesize;
 
 public:
 	XAudio2_7();
-	virtual ~XAudio2_7();
+	~XAudio2_7() override;
 
 	bool Start() override;
 	void Stop() override;
-
 	void Clear(bool mute) override;
 	void SetVolume(int volume) override;
 
 	static bool isValid() { return InitLibrary(); }
+
+#else
+
+public:
+	XAudio2_7()
+	{}
```

Comment:
This function should not be called. IMO it's better to just raise an error om compilation instead.